### PR TITLE
add alternative version to pages

### PR DESCRIPTION
**Pull request by shawnthompson**
fix capitalization in English title

### CODE REVIEW

File: src/fr/standards/index.md [hunk 1]
```diff
@@ -5,3 +5,13 @@ description: "La Division de la politique numérique du Secrétariat du Conseil
 ---
 
 {% include "partials/standard-alert.njk" %}
+
+<div class="row wb-eqht mrgn-tp-lg">
+
+  <div class="col-md-6">
+    <h2 class="h3"><a href="./what_we_heard_report">Rapport sur ce que nous avons entendu : Norme d’accessibilité des technologies de l’information et des communications (NATIC)</a></h2>
```
Comment (duboisp) [inline src/fr/standards/index.md:12]:
```suggestion
    <h2 class="h3"><a href="./what_we_heard_report">Rapport sur ce que nous avons entendu : Norme d’accessibilité des technologies de l’information et des communications (<abbr>NATIC</abbr>)</a></h2>
```

File: src/fr/standards/what_we_heard_report.md [hunk 2]
```diff
@@ -4,22 +4,48 @@ layout: layouts/base.njk
 description: "Entre le 31 octobre et le 30 novembre 2022, la Division de la politique numérique et de la performance du Secrétariat du Conseil du Trésor du Canada - Bureau du dirigeant principal de l'information (<abbr>SCT-BDPI</abbr>) a entrepris un engagement ciblé sur la <em>Norme d'accessibilité des technologies de l'information et de la communication (<abbr>TIC</abbr>)</em>. Cette page détaille les principaux thèmes qui ont émergé et les prochaines étapes à suivre en ce qui concerne la norme."
 ---
 
-## Objectif 
+<div class="row">
+
+  <div class="col-md-4 col-md-push-8 mrgn-tp-lg">
+    <a class="gc-dwnld-lnk" href="/docs/sitka-wwhr-fr.pptx">
```
Comment (duboisp) [inline src/fr/standards/what_we_heard_report.md:10]: Commentaire:
Il semble que le pattern du download link wrap le tout. Il va falloir qu'il soit revisé.

File: src/en/standards/index.md [hunk 3]
```diff
@@ -5,3 +5,12 @@ description: "The Digital Policy Division at the Treasury Board of Canada Secret
 ---
 
 {% include "partials/standard-alert.njk" %}
+
+<div class="row wb-eqht mrgn-tp-lg">
+
+  <div class="col-md-6">
+    <h2 class="h3"><a href="./what_we_heard_report">What we heard report: Standard on Information and Communication Technology Accessibility (SICTA)</a></h2>
```
Comment (duboisp) [inline src/en/standards/index.md:12]:
```suggestion
    <h2 class="h3"><a href="./what_we_heard_report">What we heard report: Standard on Information and Communication Technology Accessibility (<abbr>SICTA</abbr>)</a></h2>
```

File: src/_images/doc.png
Comment (duboisp) [inline src/_images/doc.png:1]: Commentaire:
Ça l'air qu'on n'a pas cette image sur le CDN de Canada.ca. Il va falloir qu'on l'ajoute lors de la revision du pattern.

**Review by duboisp — CHANGES_REQUESTED** [review]
Changes that need to be fixed:
* [x] Few small request change to ensure consistency through the web page. Like the use of `<abbr>` element.
* [x] The powerpoint said March 2023 and this document don't have any reference to March 2023.

Note:
I quickly reviewed the content on of the English powerpoint with the HTML page and I confirm the HTML is an equivalent version. I didn't validated the French, but I am going to assume it is the same.

File: src/fr/standards/what_we_heard_report.md (hunk 2 above)
Comment (shawnthompson) [reply src/fr/standards/what_we_heard_report.md:10]: I kind of like it like that but it look like the image and the text is a separate link.

Comment (duboisp) [reply src/fr/standards/what_we_heard_report.md:10]: Yeah but by definition it is not optimal, we can discuss offline about it.

File: src/_images/doc.png
Comment (shawnthompson) [reply src/_images/doc.png:1]: But it's in the dam and not the GCWeb folder.

**Review by duboisp — CHANGES_REQUESTED** [review]
One small changes, missing an "r" in a tag.

Regarding the abbr, some have title some don't. But we can fix that later.

**Pull request merged**